xds/resolver/watch_service.go

```diff
@@ -30,7 +30,10 @@ import (
 )
 
 import (
-	dubboLogger "dubbo.apache.org/dubbo-go/v3/common/logger"
+	dubboLogger "github.com/dubbogo/gost/log/logger"
```

Comment:
pla rename its name to 'dubbogoLogger'.  pls change all codes in other places.

Reply:
OK👌